### PR TITLE
Task2: Ivashkevich Maksim

### PR DESCRIPTION
**Pull request by zeydlitz**

### CODE REVIEW

**Review by PTaHHHa — COMMENTED** [review]
Not sure about indentations before JOIN clauses. The rest looks good.

File: Tasks/Task2/query04.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+--Query  4
+SELECT person.BusinessEntityID,
+       person.LastName,
+       person.FirstName
+FROM Person.Person AS person
+LEFT JOIN Person.BusinessEntityAddress as bea
+       ON bea.BusinessEntityID = person.BusinessEntityID
+WHERE bea.BusinessEntityID IS NULL
+      OR 
+      bea.AddressTypeID <> 2
```
Comment (YauheniD) [inline Tasks/Task2/query04.sql]: I know it's Home but what if it would changed :)

File: Tasks/Task2/query06.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+       Sales.SalesOrderHeader.OnlineOrderFlag
+FROM Sales.SalesOrderHeader
+LEFT JOIN Sales.SalesOrderDetail
+       ON Sales.SalesOrderHeader.SalesOrderID = Sales.SalesOrderDetail.SalesOrderID
+LEFT JOIN Production.Product as product
+       ON Sales.SalesOrderDetail.ProductID = product.ProductID
+WHERE Sales.SalesOrderHeader.OnlineOrderFlag = 1
+  AND Sales.SalesOrderHeader.OrderDate = CONVERT(DATETIME, '20130101', 101)
+  AND EXISTS
+    (SELECT *
```
Comment (YauheniD) [inline Tasks/Task2/query06.sql]: probably it was done for test reasons 
however  would it grant any performance increase?

File: Tasks/Task2/query06.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+--Query  6
+SELECT DISTINCT
+       Sales.SalesOrderHeader.CustomerID,
+       product.Name as BikeName,
+       Sales.SalesOrderHeader.OrderDate,
+       Sales.SalesOrderHeader.OnlineOrderFlag
+FROM Sales.SalesOrderHeader
```
Comment (YauheniD) [inline Tasks/Task2/query06.sql]: Nit: would be nice to use alias for tables to keep code easier to read

**Pull request closed without merging**